Automattic/wp-calypso
# Domains: Fix notice for expired domains that are in active auction

#### Proposed Changes

Note: Depends on D84964-code.

There is a problem in the notice shown for domains that are in aftermarket auction - it currently shows "invalid date" where the auction's end date should be:

![Markup on 2022-07-29 at 00:24:13](https://user-images.githubusercontent.com/5324818/181676650-8eeb37d6-6dbc-4f54-967b-ccdaac2de261.png)

This PR adds the aftermarket auction start and end dates to the domain object and shows the correct date for domains that are currently in auction:

![Markup on 2022-07-28 at 20:00:28](https://user-images.githubusercontent.com/5324818/181651195-bd5b01be-9227-4978-b3fb-04cce051a375.png)

#### Testing Instructions

- Apply D84964-code to your sandbox if it isn't deployed yet and sandbox `public-api`
- Open the live Calypso link or build this branch locally
- Find a domain that is currently in active auction
- Ensure the notice in the domains list shows the correct auction end date as in the screenshot above
